pixiv_dl/downloader.py

```diff
@@ -756,19 +756,22 @@ def main():
     aapi = pixivpy3.AppPixivAPI()
     aapi.set_accept_language("en-us")
     cprint("Authenticating with Pixiv...", "cyan")
-    token_file = output / "refresh_token"
-    if token_file.exists():
-        aapi.auth(refresh_token=token_file.read_text())
-        cprint(f"Successfully logged in with token as {aapi.user_id}", "green")
-    else:
-        if not args.username or not args.password:
-            cprint("No refresh token found and no username/password provided cannot login", "red")
-            return
 
+    token_file = output / "refresh_token"
+    if args.subcommand == "auth":
         aapi.auth(username=args.username, password=args.password)
         public_api.set_auth(aapi.access_token, aapi.refresh_token)
```

Comment:
This should be outside of the auth flow because it'll never get set if logging in with a token